### ISSUE TITLE
Election results: show votes per candidate in sidebar tab

### ISSUE THREAD

**Issue by traumschule** [Joystream/pioneer#4092]
On https://pioneerapp.xyz/#/election > `Voting Results` the intuitive expectation when clicking on a candidate is to see revealed votes for this candidate in a tab in the sidebar.

**Comment by dmtrjsg** [comment]
Only applies for revealing stage..but don't you get to the full screen where revealed votes are shown any ways on a full page?

**Comment by traumschule** [comment]
No, that view doesn't exist. We also want to see votes per candidate for https://pioneerapp.xyz/#/election/past-elections

**Comment by dmtrjsg** [comment]
Seem to have been done already:

<img width="1206" alt="Screenshot 2023-01-30 at 09 36 22" src="https://user-images.githubusercontent.com/93924525/215440717-ab42097a-37e1-4aaa-a390-1204356f2559.png">

<img width="510" alt="Screenshot 2023-01-30 at 09 35 57" src="https://user-images.githubusercontent.com/93924525/215440732-088f340f-9fdb-4ff6-9e98-5bd1bd271189.png">

**Comment by traumschule** [comment]
You misunderstand. It shows the number but not the voters with stakes.

**Comment by dmtrjsg** [comment]
But we cannot show the votes per candidate in the voting stage, the information is not available. We can only show total amount of people voted in general... I don't think this is what you are asking here for so seems like this request is impossible to implement 😞

**Comment by traumschule** [comment]
It should be doable for revealing stage and past elections, no?

**Comment by dmtrjsg** [comment]
Sorry for probably stupid question but how is [that >](https://github.com/Joystream/pioneer/issues/4092#issuecomment-1408270872)  not achieving the same? Or are you talking about showing each individual vote with a corresponding stake?

**Comment by traumschule** [comment]
yes

**Comment by dmtrjsg** [comment]
Gotcha!

⚠️  designer who is taking this on: 
We can use inspiration for design from this issue see [figma link](https://www.figma.com/file/YBfvoRpnduOXxUnA9kvNP5/Pioneer-Design?t=x8iA8GtTgdY9NMr5-0) 

- #2365 

Bear in mind For some votes we can show members, for others it will be unknown addresses.
